calibration_tools_standalone.repos

```diff
@@ -0,0 +1,49 @@
+repositories:
+  calibration_tools:
+    type: git
+    url: https://github.com/knzo25/CalibrationTools.git
```

Comment:
knzo25 to tier4

Reply:
Thanks. Addressed in [6820271](https://github.com/tier4/CalibrationTools/pull/164/commits/68202717d7c18f4444fd41b3557f02ac9049ea17)